src/platform/user/profile/vet360/components/base/Vet360EditModal.jsx

```diff
@@ -19,7 +19,7 @@ export default class Vet360EditModal extends React.Component {
     hasValidationError: PropTypes.func,
     isEmpty: PropTypes.bool.isRequired,
     onCancel: PropTypes.func.isRequired,
-    onChange: PropTypes.func.isRequired,
+    onChangeFormDataAndSchemas: PropTypes.func.isRequired,
```

Comment:
is this something I missed in my code cleanup earlier this week?

Reply:
@erikphansen Yeah it was causing a console error

Reply:
thanks